code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -26,17 +26,18 @@
 	ventcrawler = VENTCRAWLER_ALWAYS
 	unique_name = TRUE
 	faction = list("rat")
+	///The spell that the rat uses to scrounge up junk.
 	var/datum/action/cooldown/coffer
+	///The Spell that the rat uses to recruit/convert more rats.
 	var/datum/action/cooldown/riot
-	///Number assigned to rats and mice, checked when determining infighting.
 
 /mob/living/simple_animal/hostile/regalrat/Initialize()
 	. = ..()
 	coffer = new /datum/action/cooldown/coffer
 	riot = new /datum/action/cooldown/riot
 	coffer.Grant(src)
 	riot.Grant(src)
-	INVOKE_ASYNC(src, .proc/get_player)
+	AddElement(/datum/element/waddling)
```

Comment:
lol

Reply:
Honestly, since the regal rat sprites were changed to look more like a kangaroo rat than an American Grey rat, and being that it now stands on it's hindlegs, it works thematically and visually.